ui/app/AppLayouts/Wallet/panels/SwapInputPanel.qml

```diff
@@ -39,33 +39,40 @@ Control {
     onTokenKeyChanged: Qt.callLater(reevaluateSelectedId)
 
     property string tokenAmount
-    onTokenAmountChanged: Qt.callLater(d.updateInputText)
+    onTokenAmountChanged: Qt.callLater(d.updateInputText) // FIXME remove the callLater(), shouldn't be needed now
 
     property int swapSide: SwapInputPanel.SwapSide.Pay
     property bool fiatInputInteractive
     property bool mainInputLoading
     property bool bottomTextLoading
     property bool interactive: true
 
+    // FIXME drop after using ModelEntry, shouldn't be needed
```

Comment:
any GH issues created fir FIXME's?

Reply:
No, it was mostly for me (and you 🙂), regarding the TokenSelector